### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by dashaivasha**

### CODE REVIEW

File: Boosting/PageObjects/HomePage.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            SignInButton.SendKeys(Keys.Control + Keys.Enter);
+        }
+
+        public void SwitchToNewTab()
+        {
+            for (int i=0; i < GetTabsId();i++)
+            { 
+                Driver.SwitchTo().Window(Driver.WindowHandles[i]);
+                Driver.Close();
+
```
Comment (Dalalaler) [inline Boosting/PageObjects/HomePage.cs:32]: Watch your code.

File: Boosting/PageObjects/HomePage.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            }
+        }
+
+        public int GetTabsId()
+        {
+            List<String> tabs_id = new List<String>(Driver.WindowHandles);
+            return tabs_id.Count;
```
Comment (Dalalaler) [inline Boosting/PageObjects/HomePage.cs:39]: Missed blank line.

File: Boosting/PageObjects/HomePage.cs [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+
+namespace Boosting.PageObjects
+{
+    public class HomePage : BasePage
+    {
+        private By MyCartButton = By.XPath("//div[@class='shopping_cart']//a[contains(@title,'View')]");
+        private IWebElement SignInButton => Driver.FindElement(By.XPath("//div[@class='header_user_info']//a"));
```
Comment (Dalalaler) [inline Boosting/PageObjects/HomePage.cs:11]: Fix with naming rules.

File: Boosting/PageObjects/HomePage.cs [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        
+        public HomePage()
+        {
+        }
+
+        public void OpenCartInNewTab()
+        {
+            WebDriverExtensions.FindElement(Driver, MyCartButton, Data.WaitTime).SendKeys(Keys.Control + Keys.Enter);
+        }
+        public void OpenSignInNewTab()
```
Comment (Dalalaler) [inline Boosting/PageObjects/HomePage.cs:21]: Blank line.

File: Boosting/PageObjects/DemoQa.cs [hunk 5]
```diff
@@ -0,0 +1,37 @@
+﻿using Boosting.Driver;
+using OpenQA.Selenium;
+
+namespace Boosting.PageObjects
+{
+    internal class DemoQa : BasePage
```
Comment (Dalalaler) [inline Boosting/PageObjects/DemoQa.cs:6]: DemoQa what? Give acceptable name.

File: Boosting/PageObjects/DemoQa.cs [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+        public void Switch()
+        {
+            for (int i = 0; i < GetIframes(); i++)
+            {
+                Driver.SwitchTo().Frame(i);
+            }
+        }
+
+        public bool IsItDefaultContent()
```
Comment (Dalalaler) [inline Boosting/PageObjects/DemoQa.cs:32]: Use lambda here.

File: Boosting/PageObjects/DemoQa.cs [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+namespace Boosting.PageObjects
+{
+    internal class DemoQa : BasePage
+    {
+        public DemoQa()
+        {
+        }
+
+        public int GetIframes()
```
Comment (Dalalaler) [inline Boosting/PageObjects/DemoQa.cs:12]: Wrong method name.

File: Boosting/PageObjects/DemoQa.cs [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+            return iframeElements.Count;
+        }
+
+        public void SwitchToDefaultContent()
+        {
+            Driver.SwitchTo().DefaultContent();
+        }
+
+        public void Switch()
```
Comment (Dalalaler) [inline Boosting/PageObjects/DemoQa.cs:24]: Switch where? Give acceptable name for method.

File: Boosting/Driver/DriverFactory.cs [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+        {
+            _webDriver?.Quit();
+            _webDriver?.Dispose();
+            _webDriver = null;
+        }
+
+        public static void InitalizerDriver()
+        {
+            var browser = Data.Browser;
+            Enum.TryParse(browser, out BrowserNameEnum browserNameEnum);
```
Comment (Dalalaler) [inline Boosting/Driver/DriverFactory.cs:41]: Please, explain meaning of this string of code.

File: Boosting/Tests/UITests/BaseTest.cs [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+using Boosting.PageObjects;
+using Boosting.TestData;
+using System.Net.Http;
+
+namespace Boosting.Tests.UiTests
+{
+    public class BaseTest
+    {
+        protected TestDetails Data = JsonManager.GetTestData();
+        internal HomePage homePage = new HomePage();
```
Comment (Dalalaler) [inline Boosting/Tests/UITests/BaseTest.cs:13]: Try not to use internal modifiers in Base and another test classes.

File: Boosting/Tests/UITests/IFramesTest.cs [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+    internal class IFramesTest : BaseTest
+    {
+        [SetUp]
+        public void Open()
+        {
+            WebDriver.Navigate().GoToUrl(Data.Url);
+        }
+
+        [Test]
+        public void OpenTwoTabs()
```
Comment (Dalalaler) [inline Boosting/Tests/UITests/IFramesTest.cs:16]: Why is this method called OpenTwoTabs? What is the difference between iFrames and And Tabs? Also, it is very and very bad practice to have two tests with the same name.

File: Boosting/PageObjects/HomePage.cs (hunk 1 above)
Comment (dashaivasha) [reply Boosting/PageObjects/HomePage.cs:32]: resolved

File: Boosting/PageObjects/HomePage.cs (hunk 2 above)
Comment (dashaivasha) [reply Boosting/PageObjects/HomePage.cs:39]: resolved

File: Boosting/PageObjects/HomePage.cs (hunk 3 above)
Comment (dashaivasha) [reply Boosting/PageObjects/HomePage.cs:11]: resolved

File: Boosting/PageObjects/DemoQa.cs (hunk 5 above)
Comment (dashaivasha) [reply Boosting/PageObjects/DemoQa.cs:6]: resolved

File: Boosting/PageObjects/DemoQa.cs (hunk 8 above)
Comment (dashaivasha) [reply Boosting/PageObjects/DemoQa.cs:24]: resolved

File: Boosting/Tests/UITests/BaseTest.cs (hunk 10 above)
Comment (dashaivasha) [reply Boosting/Tests/UITests/BaseTest.cs:13]: resolved

File: Boosting/Tests/UITests/IFramesTest.cs (hunk 11 above)
Comment (dashaivasha) [reply Boosting/Tests/UITests/IFramesTest.cs:16]: resolved

**Pull request merged**